README.md

```diff
@@ -137,6 +137,10 @@ Supported:
 * [yarn](#yarn)
 * [bundler](#bundler)
 
+Experimental:
+
+* [generic fetcher](#generic-fetcher)
```

Comment:
> Isn't just the word generic enough here ? Since that's what is used in the CLI.

>> I think Github didn't link your comment to the line you were refering to.

Here for example (and other places)

Reply:
I think `generic` by itself does not mean anything really, that's why the fetcher part.